src/patternfly/components/Dropdown/styles.scss

```diff
@@ -0,0 +1,158 @@
+$pf-c-dropdown__menu-item--PaddingBottom: $pf-global--spacer--sm !default;
+$pf-c-dropdown__menu-item--PaddingLeft: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--first--PaddingTop: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--last--PaddingBottom: $pf-global--spacer--md !default;
+$pf-c-dropdown__menu-item--Color: $pf-global--Color--dark-100 !default;
+$pf-c-dropdown__menu-item--is-disabled--Color: $pf-global--Color--dark-200 !default;
+
+$pf-c-dropdown__menu-item--hover--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+// Menu Item Separator
+$pf-c-dropdown__separator--Height: $pf-global--BorderWidth--md !default;
+$pf-c-dropdown__separator--BackgroundColor: $pf-global--BackgroundColor--light-300 !default;
+
+
+.pf-c-dropdown {
+  position: relative;
+  display: inline-block;
+
+  &__toggle {
+    padding: $pf-c-dropdown__toggle--PaddingTop $pf-c-dropdown__toggle--PaddingRight $pf-c-dropdown__toggle--PaddingBottom $pf-c-dropdown__toggle--PaddingLeft;
+    font-size: $pf-c-dropdown__toggle--FontSize;
+    color: $pf-c-dropdown__toggle--Color;
+    background-color: $pf-c-dropdown__toggle--BackgroundColor;
+    border: $pf-c-dropdown__toggle--BorderWidth solid $pf-c-dropdown__toggle--BorderColor;
+
+    &:hover {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-bottom: calc(#{$pf-c-dropdown__toggle--PaddingBottom} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--hover--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-width: $pf-c-dropdown__toggle--hover--BorderWidth;
+    }
+
+    .pf-is-expanded & {
+      padding-top: calc(#{$pf-c-dropdown__toggle--PaddingTop} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-right: calc(#{$pf-c-dropdown__toggle--PaddingRight} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-bottom: calc(#{$pf-c-dropdown__toggle--PaddingBottom} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      padding-left: calc(#{$pf-c-dropdown__toggle--PaddingLeft} - (#{$pf-c-dropdown__toggle--is-expanded--BorderWidth} - #{$pf-c-dropdown__toggle--BorderWidth}));
+      border-color: $pf-c-dropdown__toggle--is-expanded--BorderColor;
+      border-width: $pf-c-dropdown__toggle--is-expanded--BorderWidth;
+    }
+
+    .pf-is-ghost & {
+      background-color: transparent;
+      border-color: transparent;
+    }
+
+    .pf-sans-arrow & {
```

Comment:
Is `sans` another modifier type? Or should `is` be always used?

Reply:
I believe @andresgalante agreed that PF should stick to a single consistent naming scheme for all modifiers. Having multiple variants for modifiers makes it difficult for creating utilities that can auto prefix modifier states. It would need too much smarts to know when to use `is` vs `has` vs `sans`.
`pf-is-noarrow` seems ok to me.

Reply:
That's a great point. I like `pf-is-noarrow`. Thanks for the suggestion!

Reply:
I don't like `sans` either. 

@christianvogt I want to build modifier with both `-is` or `-has`.